opm/grid/cpgpreprocess/preprocess.c

```diff
@@ -802,16 +980,16 @@ reverse_face_nodes(struct processed_grid *out)
 }
 
 
-/*-----------------------------------------------------------------
-  Public interface
-*/
-void process_grdecl(const struct grdecl   *in,
-                    double                tolerance,
-                    const int*     is_aquifer_cell,
-                    struct processed_grid *out,
-                    int pinchActive)
+/* ----------------------------------------------------------------------
+ * Public interface
+ * ---------------------------------------------------------------------- */
+int process_grdecl(const struct grdecl   *in,
+                   double                 tolerance,
+                   const int             *is_aquifer_cell,
+                   struct processed_grid *out,
+                   int                    pinchActive)
 {
-    struct grdecl g;
+    struct grdecl g = {0};
```

Comment:
Out of curiosity: Is this valid C?

Reply:
> Is this valid C?

Yes.  It's the portable way of zeroing out an object of `struct` type.  Integers will be `0`, floating-point values will be `0.0` and pointers will be `NULL`.  That may not what's needed in every case, but it is what I want here.